test/data/stages/firewall/diff.json

```diff
@@ -2,7 +2,8 @@
   "added_files": [
     "/etc/firewalld/firewalld.conf.old",
     "/etc/firewalld/zones/trusted.xml",
-    "/etc/firewalld/zones/trusted.xml.old"
+    "/etc/firewalld/zones/trusted.xml.old",
+    "/etc/firewalld/zones/new-zone.xml"
```

Comment:
The tests fail on my local fork just for this stage. I wasn't sure if it was because we need both `/etc/firewalld/zones/new-zone.xml` and `/etc/firewalld/zones/new-zone.xml.old` (my first approach) or not. But both approaches fail. 
I haven't a clue why the one with both files (the approach that makes sense to me) does.

Reply:
I've also completely removed them in case that a reload was necessary, but same issue.

Reply:
You current PR has an empty diff, that certainly does not work since the zone-configuration will somehow end up on disk, right? Do you have a link to a failed run so we can look at the error report?

Reply:
The empty diff is a slip on my end.

- Test failing with both `new-zone.xml` and `new-zone.xml.old` added: [CI run](https://github.com/7flying/osbuild/actions/runs/3310858532)
- Test failing with just `new-zone.xml`: [CI run](https://github.com/7flying/osbuild/actions/runs/3311329209)

Reply:
Uff, for some reason the diff is not printed in CI, and instead it has an error about being unable to access the tree directories. Maybe @gicmo has an idea?

Reply:
I will ask on the slack channel